integration-templates/sharepoint-online/nango.yaml

```diff
@@ -36,6 +36,7 @@ integrations:
                 version: 1.0.0
 models:
     FileMetadata:
+        siteId: string
```

Comment:
You should bump the patch version of the sync this affects

Reply:
Thanks @khaliqgant  for clarifying this 🙏